doc/rados/operations/pools.rst

```diff
@@ -134,31 +137,42 @@ following:
 .. describe:: {pg-num}
 
    The total number of PGs in the pool. For details on calculating an
-   appropriate number, see :ref:`placement groups`. The default value ``8`` is
+   appropriate number, see :ref:`placement groups`. The default value is
    NOT suitable for most systems.
 
   :Type: Integer
   :Required: Yes.
-  :Default: 8
+  :Default: 32
 
 .. describe:: {pgp-num}
 
    The total number of PGs for placement purposes. This **should be equal to
    the total number of PGs**, except briefly while ``pg_num`` is being
-   increased or decreased. 
+   increased or decreased. Note that in releases beginning with Nautilus one
+   generally does not change ``pgp_num`` directly:  Ceph will automatically and
+   incrementally scale ``pgp_num`` for a given pool when ``pg_num`` for that pool
+   has been adjusted.  Adjustments to ``pg_num`` may be either made by the
+   PG autoscaler, or if the autoscaler is disabled for a given pool by a manual
+   setting via the CLI or dashboard.
 
   :Type: Integer
   :Required: Yes. If no value has been specified in the command, then the default value is used (unless a different value has been set in Ceph configuration).
-  :Default: 8
+  :Default: 32
 
 .. describe:: {replicated|erasure}
 
-   The pool type. This can be either **replicated** (to recover from lost OSDs
-   by keeping multiple copies of the objects) or **erasure** (to achieve a kind
-   of `generalized parity RAID <../erasure-code>`_ capability).  The
-   **replicated** pools require more raw storage but can implement all Ceph
-   operations. The **erasure** pools require less raw storage but can perform
-   only some Ceph tasks and may provide decreased performance.
+   The pool's data protection strategy. This can be either ``replicated``
+   (like RAID1 and RAID10) ``erasure (a kind
+   of `generalized parity RAID <../erasure-code>`_ strategy like RAID6 but
+   more flexible).  A 
+   ``replicated`` pool yields less usable capacity for a given amount of
+   raw storage but is suitable for all Ceph components and use cases.
+   An ``erasure`` (EC) pool often yields more usable capacity than replication
+   for a given amount of underlying raw storage, but is only suitable for 
+   a subset of Ceph components and use cases.  Depending on the workload and
+   the specific profile, EC usually requires more failure domains than
+   replication and provide decreased performance, but may tolerate a greater
```

Comment:
s/provide/provides. "EC", the subject of the sentence, is singular. I think this is an agreement-with-nearest error.

Reply:
pushing